File: solutions/HJK9810/2026-09-01/medium/7102.py
def algorithm():
    N, M = map(int, input().split())
    sum_list = [0] * (N + M + 1)

    for first in range(1, N + 1):
        for second in range(1, M + 1):
            sum_list[first + second] += 1

    max_count = 0
    for num in range(len(sum_list)):
        if max_count < sum_list[num]:
            max_count = sum_list[num]

    result = []
    for num in range(len(sum_list)):
        if sum_list[num] == max_count: result.append(str(num))

    return " ".join(result)

T = int(input())
# 여러개의 테스트 케이스가 주어지므로, 각각을 처리합니다.
for test_case in range(1, T + 1):
    print(f"#{test_case} {algorithm()}")

File: solutions/HJK9810/2026-09-01/medium/7102.java
import java.util.Scanner;

class Solution {
	public static void main(String args[]) throws Exception {
		Scanner sc = new Scanner(System.in);
    StringBuilder sb = new StringBuilder();
    int T = sc.nextInt();

		for(int test_case = 1; test_case <= T; test_case++) {
      sb.setLength(0);
      int N = sc.nextInt();
      int M = sc.nextInt();

      int[] counts = new int[N + M + 1];
      for (int first = 1; first <= N; first++) {
        for (int second = 1; second <= M; second++) {
          counts[first + second]++;
        }
      }

      int maxValue = counts[0];
      for (int num = 2; num < N + M + 1; num++) {
        if (maxValue < counts[num]) {
          maxValue = counts[num];
          sb.setLength(0);
          sb.append(" " + num);
        } else if (maxValue == counts[num]) {
          sb.append(" " + num);
        }
      }

      System.out.println("#" + test_case + sb.toString());
		}
    sc.close();
	}
}
